src/Core/Prefab/GlobeView.js

```diff
@@ -250,9 +250,9 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
         if (len < lim) {
             var t = Math.pow(Math.cos((lim - len) / (lim - v.x * size * 0.9981) * Math.PI * 0.5), 1.5);
             var color = new THREE.Color(0x93d5f8);
-            this.mainLoop.gfxEngine.renderer.setClearColor(color.multiplyScalar(1.0 - t));
+            this.mainLoop.gfxEngine.renderer.setClearColor(color.multiplyScalar(1.0 - t), options.alphaClearValue);
```

Comment:
This is a new feature in a `add example`commit. I should live in a separate PR or at the minimum in a separate commit.